### PR TITLE
Updated Peewee Recipe.

### PR DESCRIPTION
**Pull request by dharhas**
This rebases #847 to latest master and removes dependency on gcc.

### CODE REVIEW

File: recipes/peewee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: http://github.com/coleifer/peewee/
+  license: MIT
+  summary: 'A little ORM'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
+    - dharhas
```
Comment (ocefpaf) [inline recipes/peewee/meta.yaml]: :+1:

File: recipes/peewee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  build:
+    - python
+    - cython >=0.22.1
+
+  run:
+    - python
+
+test:
+  imports:
+    - playhouse
```
Comment (ocefpaf) [inline recipes/peewee/meta.yaml]: @dharhas I am not a `peewee` user so I am not sure if that is the only import worth testing. Can you check that?

PS: It seems we have a successful build :tada:  I will merge this as soon.

**Comment by dharhas** [comment]
I mainly use playhouse, but I added a couple of other imports that seemed like they covered core functionality.

File: recipes/peewee/meta.yaml [hunk 3]
```diff
@@ -28,6 +28,13 @@ requirements:
 test:
   imports:
     - playhouse
+    - SqliteDatabase
+    - Model
+    - Query
+    - Field
+    - Proxy
+    - transaction
+    - Database
```
Comment (ocefpaf) [inline recipes/peewee/meta.yaml]: All those modules are part of `peewee`?

Comment (ocefpaf) [inline recipes/peewee/meta.yaml]: My original question was just b/c I was puzzled with the package name vs the import name (`peewee` vs `playhouse`).

Looking at the repo it seems that `playhouse` is the right test though.

**Comment by dharhas** [comment]
my mistake... these are all from peewee import blah type cases except for peewee.

**Comment by dharhas** [comment]
so peewee is the main package, playhouse is a bunch of special extensions.

**Comment by ocefpaf** [comment]
As soon we have green we can merge this. Thanks @dharhas.

PS: @pmlandwehr can you take a quick look at the recipe and give your :+1:

**Comment by ocefpaf** [comment]
We have green! Thanks @dharhas and @pmlandwehr!

**Comment by dharhas** [comment]
thanks. @ocefpaf how longish before this shows up in the actual conda-forge channel?

**Comment by ocefpaf** [comment]
> how longish before this shows up in the actual conda-forge channel?

That varies a lot depending on the CIs. The feedstock is already here:

https://github.com/conda-forge/peewee-feedstock

And I believe Linux and OS X already completed the builds:

http://anaconda.org/conda-forge/peewee/files

Windows might take a while:

https://ci.appveyor.com/project/conda-forge/peewee-feedstock/branch/master

**Comment by jakirkham** [comment]
Well this converted ok which is a huge success in my book given the issues with Travis CI of late. I'm chalking this up in the win category.

**Comment by jakirkham** [comment]
@coleifer, just letting you know we have added `peewee` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge peewee`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/peewee-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `peewee`. 😄

**Comment by coleifer** [comment]
Thanks for your efforts adding this to your project, glad you've found peewee useful.

**Pull request merged**